### PR TITLE
Proposed minor upgrades

### PR DESCRIPTION
**Pull request by koriaf**
clean and make more detailed explanation of tap-gw and tap relationships, add examples

### CODE REVIEW

File: docs/1.0/index.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ ausdigital-idp/1 | Version 1 of the [AusDigtial](http://ausdigital.org) [IDP](ht
 
 ## Licence
 
-Copyright (c) 2016 the Editor and Contributors. All rights reserved.
+Copyright (c) 2017 the Editor and Contributors. All rights reserved.
```
Comment (monkeypants) [inline docs/1.0/index.md]: I think this should be "...(c) 2016, 2017..."

File: docs/1.0/index.md [hunk 2]
```diff
@@ -121,58 +121,59 @@ interpreted as described in RFC 2119.
...
 
 These endpoints are identified with a URL, comprised of an the TAP-GW provider's
 DNS domain followed by an RFC 4122 Universally Unique Identifier (UUID).
 
 For example:
-```
-https://tap.testpoint.io/886313e1-3b8a-5372-9b90-0c9aee199e5d/
-```
+* `https://tap.testpoint.io/886313e1-3b8a-5372-9b90-0c9aee199e5d/`, or
+* `https://my-invoice-provider.com/api/tap/Z2o6mR51EME/`, or any other URL
```
Comment (monkeypants) [inline docs/1.0/index.md]: my-access-point.com

File: docs/1.0/index.md [hunk 3]
```diff
@@ -121,58 +121,59 @@ interpreted as described in RFC 2119.
...
 
 This endpoint does not directly reveal any information about the message
 receiving party, however the `ausdigital-dcp` reference to the endpoint will (if
 metadata about the endpoint has been published).
 
-TODO: as a TAP-GW user:
+Endpoint metadata - arbitrary key/values associated with the endpoints by endpoint owner for his internal needs.
```
Comment (monkeypants) [inline docs/1.0/index.md]: ...for their internal needs.

gender neutral language preferred

File: docs/1.0/index.md [hunk 4]
```diff
@@ -121,58 +121,59 @@ interpreted as described in RFC 2119.
...
-   client (the TAP-GW is the RP)?
+ * TAP-GW is a TAP protocol extension, which means that TAP-GW installation may be used as TAP (and basically emulates multiple TAP installations at same domain)
+ * base pathes are supported: specific installation may prefix all endpoints from this specification by some url
+    * for example, given we have installation `https://tap-gw.testpoint.io/` and base path is `/api/v1/`, endpoints list url will be available at `https://tap-gw.testpoint.io/api/v1/endpoints/`
+ * TAP-GW SHOULD support JWT user authentication
+ * TAP-GW MAY support any other user auth method
 
 TAP-GW customers can create an arbitrary number of endpoints, however these
-endpoints are "anonymous" until they are mapped to the integration surface of an identified business (via endpoint publishing mechanism)
+endpoints are "anonymous" until they are mapped to the integration surface of an identified business (via endpoint publishing mechanism). Endpoints may be created on behalf of customer, for example by customer's invoicing software
```
Comment (monkeypants) [inline docs/1.0/index.md]: s/customer's/participant's/

s/invoicing/ledger/

**Comment by koriaf** [comment]
The requested changes have been made

**Pull request merged**